systems/subscriber/sim-manager/pkg/db/package_repo.go

```diff
@@ -30,13 +31,15 @@ func NewPackageRepo(db sql.Db) PackageRepo {
 
 func (p *packageRepo) Add(pkg *Package, nestedFunc func(pkg *Package, tx *gorm.DB) error) error {
 	err := p.Db.GetGormDb().Transaction(func(tx *gorm.DB) error {
+		log.Info("Running nested function")
```

Comment:
This line was for debugging purposes. we can remove it.

Reply:
addressed